tests/Stub/TestATestStubSigInt.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Stub;
+
+class TestATestStubSigInt extends BrokenTestBase implements BrokenTestInterface
+{
+    public function testBrokenTest(): void
+    {
+        usleep(200000);
```

Comment:
A sleep here creates a brittle test. Maybe we could wrap this inside a method that sleeps from a smaller amount of time and retries N times before giving up?

Reply:
fixed